Cargo.toml

```diff
@@ -1,5 +1,5 @@
 [workspace]
-
+resolver = "2"
```

Comment:
`resolver = "2"` (that fixes various cargo features/workspace related bugs) is needed to check all feature combinations works properly [without editing `Cargo.toml`](https://github.com/taiki-e/cargo-hack/issues/15).